.github/actions/configure-bazel/action.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Configure Bazel
+description: Configure Bazel for OpenTitan CI Job
+
+runs:
+  using: composite
+  steps:
+    - name: Populate ~/.bazelrc
+      run: |
+        cp ci/.bazelrc ~/.bazelrc
+        # Inject the OS version into a parameter used in the action key computation to
+        # avoid collisions between different operating systems in the caches.
+        # See #14695 for more information.
+        echo "build --remote_default_exec_properties=OSVersion=\"$(lsb_release -ds)\"" >> ~/.bazelrc
```

Comment:
Hmm, I wonder if this is still an issue. Can Bazel really not use the cache between glibc versions? Does it link things like `opentitantool` against the system glibc?

Reply:
A binary built against new glibc in general cannot be used with an old libc, so we cannot use binaries built on Ubuntu 22.04 on Ubuntu 20.04, for example.

Reply:
I was hoping that Bazel's hermetic builds also included the libc, but it seems not